STSampling.h

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2019, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: Interface to the space-time sampling index selection algorithm.
+
+#ifndef included_STSAMPLING_h
+#define included_STSAMPLING_h
+
+namespace CAROM {
+
+/**
+ * @brief
+ *
+ * @param[in] s_basis The spatial basis vectors.
+ * @param[in] t_basis The temporal basis vectors.
+ * @param[in] num_samples The number of samples to compute.
```

Comment:
It seems the doxygen comment needs updating to match with the parameters of the function.

Reply:
Updated.